Core/ConfigImport.php

```diff
@@ -262,8 +262,7 @@ protected function importConfigValues($aConfigValues)
             //fall back to a empty array
             $aDisabledModules = [];
         }
-        $disabledModulesBeforeImport = array_flip($aDisabledModules);
-        $disabledModulesBeforeImport = array_flip($disabledModulesBeforeImport);
+        $disabledModulesBeforeImport = array_fill_keys($aDisabledModules, 1);
```

Comment:
Maybe this is better
```php
$disabledModulesBeforeImport = array_fill_keys($aDisabledModules, true);
```

You do this to check that later here:
```
if ($disabledModulesBeforeImport[$sModuleId]) {
```
Is that right?

Reply:
yes